src/rgw/driver/rados/rgw_object_expirer_core.cc

```diff
@@ -188,7 +188,7 @@ int RGWObjExpStore::objexp_hint_trim(const DoutPrefixProvider *dpp,
   }
   auto& ref = obj.get_ref();
   int ret = cls_timeindex_trim_repeat(dpp, ref, oid, utime_t(start_time), utime_t(end_time),
-          from_marker, to_marker);
+          from_marker, to_marker, null_yield);
```

Comment:
You added an optional_yield to `objexp_hint_trim` so you don't need a `null_yield` here.